less/home-page.less

```diff
@@ -1,32 +1,89 @@
-.home {
+// testing a commit ignore this comment
+.content-container {
+  @media @desktop {
+    width: 85%;
+  }
+}
 
-  .content-section {
-    margin: 30px 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
+h2 {
+  @media @desktop {
+    font-size: 2.5rem;
+  }
+}
+
+h4 {
+  @media @desktop {
+    font-size: 2rem;
+  }
+}
+
+p {
+  @media @desktop {
+    font-size: 1.3rem;
+  }
 
-   
-    
+}
+header {
+  padding-top: 1rem;
+  .border {
+    border-top: 2px dashed gray;
+    width: 100vw;
+    position: absolute;
+    left: 0;
+  }
```

Comment:
There is currently an issue with `100vw` wherein all browsers except Firefox incorrectly cause a horizontal scrollbar to appear because the vertical scrollbar is considered to be part of the viewport. For this reason, I would recommend sticking with your previous idea of using `width: 100%`.

More info about the `100vw` issue here: https://caniuse.com/#feat=viewport-units